libraries/hatch-server-user-management/src/UserInfoRequest.ts

```diff
@@ -0,0 +1,77 @@
+import {BasicRouteParams} from '@launchtray/hatch-server-middleware';
+import {containerSingleton, Logger} from '@launchtray/hatch-util';
+import cookie from 'cookie';
+import {AUTH_ACCESS_TOKEN_COOKIE_NAME} from './constants';
+import AWSCognitoClient from './AWSCognitoClient';
+
+export class UserInfo {
+  public username?: string;
+  public userId?: string;
+  public token?: string;
+  public isAuthenticated: boolean = false;
+}
+
+// tslint:disable-next-line:max-classes-per-file
```

Comment:
Let's pull UserInfo out of this class to avoid this exception

Reply:
Looks like it's defined in UserServiceClient, too

Reply:
Updated